keyboards/jones/v.0.3.1/matrix.c

```diff
@@ -0,0 +1,234 @@
+/*
```

Comment:
Could this be converted to the "lite" version of custom matrix? 

https://docs.qmk.fm/#/custom_matrix?id=lite

Reply:
Converted to "lite" version.